frontend/src/routes/manage/Realm/Content/AddButtons.tsx

```diff
@@ -150,6 +150,7 @@ export const AddButtons: React.FC<Props> = ({ index, realm }) => {
                         label={t("manage.realm.content.add-video")}
                         onClick={() => addBlock("Video", (_store, block) => {
                             block.setValue(true, "showTitle");
+                            block.setValue(false, "showLink");
```

Comment:
Mhh good question about a good default. Of course this is the most-backwards-compatible solution. But I always like to use these situations to nudge users towards what I think is better :smile: As a user, I'd like to always be able to go the video page of an embedded video. 

So mh wait, what is even the reasoning for us to make it configurable instead of always showing a link? Probably users who want to dunno, create a blog like page with super short videos embedded, that don't make sense without context. Then it probably makes no sense to have this link.

So mh, I guess I'm saying is: `false` -> `true`? Opinions?

Reply:
Sure, I wouldn't mind that.